content/en/blog/2022/K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: blog
+title: "K8s Triage bot Helper CI Job: ‘*make update*’"
+date: 2022-03-15
```

Comment:
The actual date stands at 2022-02-25, should it be changed?

Reply:
This should be a date in the future. Ideally, co-ordinate that date with the [k/website](https://github.com/kubernetes/website/) blog team (eg: talk to me) as well so that it goes out on a day when no k/website PRs are due.